x/staking/keeper/params.go

```diff
@@ -20,26 +20,42 @@ func ParamKeyTable() params.KeyTable {
 
 // UnbondingTime
 func (k Keeper) UnbondingTime(ctx sdk.Context) (res time.Duration) {
-	k.paramstore.Get(ctx, types.KeyUnbondingTime, &res)
+	err := k.paramstore.Get(ctx, types.KeyUnbondingTime, &res)
+	if err != nil {
+		// TODO: return error - needs rewrite interfaces
+		// and handle error on the caller side
+	}
 	return
 }
 
 // MaxValidators - Maximum number of validators
 func (k Keeper) MaxValidators(ctx sdk.Context) (res uint16) {
-	k.paramstore.Get(ctx, types.KeyMaxValidators, &res)
+	err := k.paramstore.Get(ctx, types.KeyMaxValidators, &res)
```

Comment:
```suggestion
	if err := k.paramstore.Get(ctx, types.KeyMaxValidators, &res); err != nil {
```

Same goes for all other cases below